lib/js/jumpstart_ui.behavior.js

```diff
@@ -15,11 +15,20 @@ export default {
       $('figure .media-entity-wrapper.video', context).each((i, video) => {
         $(video).closest('figure').css('width', '100%');
       });
+
+      // Changing the skip link from an anchor to a div.
+      const skiplinktarget = $('a#main-content');
+      const skiplinktargetContent = $(skiplinktarget).text();
+
+      skiplinktarget.wrap('<div id="main-content"></div>');
+      $('div#main-content').attr('tabindex', '-1')
+        .addClass('visually-hidden focusable').append(skiplinktargetContent);
+      skiplinktarget.remove();
```

Comment:
All of this js work can actually be done in PHP. Take a look to see if you can make it work in that direction. It'll give you a good way to learn more about render arrays.
see the [block code](https://github.com/SU-SWS/jumpstart_ui/blob/8.x-1.x/src/Plugin/Block/SkipNavAnchorBlock.php)
see the [html tag documentation](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21Element%21HtmlTag.php/class/HtmlTag/8.2.x)

Let me know if you aren't sure about it.

Reply:
Thank you @pookmish. That was exactly what I wanted to know. I think it belongs in our JumpstartUI, right?

Reply:
Yep.

Reply:
@pookmish I got this to work with the php. 👍 I want to review with @cjwest. So I will move it to not ready for merging. Would that be the best solution?